ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -86,10 +86,23 @@
       regexFr: Regexes.startsUsing({ id: '4BAB', source: 'Ramuh', capture: false }),
       regexJa: Regexes.startsUsing({ id: '4BAB', source: 'ラムウ', capture: false }),
       regexKo: Regexes.startsUsing({ id: '4BAB', source: '라무', capture: false }),
-      infoText: {
-        en: 'Grab an orb',
-        fr: 'Prenez un orb',
-        ko: '구슬 줍기',
+      infoText: function(data) {
+        if (data.fourteenCount < 3) {
+          return {
+            en: 'Grab an orb',
+            fr: 'Prenez un orb',
+            ko: '구슬 줍기',
+          };
+        } else if (data.fourteenCount == 3) {
+          return {
+            en: 'Enrage Soon',
```

Comment:
Could you drop the enrage text? At least for consistency, nothing else shows the enrage except for Ruby ex which has weird timing where the enrage can suddenly appear.  If the enrage isn't here, then this could just be a condition on fourteenCount.

Reply:
I tried to say, there will be an enrage so just don't grab orbs and focus on dps.
should I say 'don't grab orbs'?

Reply:
Ah, I see.  Yeah, I'm not sure how to communicate all of that.  "Don't get orbs" seems like a "don't get orbs, or you'll die" kind of warning.  Maybe this is just an optimization and no need for a trigger.

I think usually I use the timeline as the "enrage" marker and not triggers, for what it's worth.

Reply:
I just removed message on last Fury's fourteen.